source/api_reference/index.html.md.erb

```diff
@@ -241,25 +241,6 @@ For example:
 }
 ```
 
-## Card types
-
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-| card_brand       | type   | label            |
-|------------------|--------|------------------|
-| visa             | DEBIT  | Visa             |
-| visa             | CREDIT | Visa             |
-| master-card      | DEBIT  | Mastercard       |
-| master-card      | CREDIT | Mastercard       |
-| maestro          | DEBIT  | Maestro          |
-| american-express | CREDIT | American Express |
-| diners-club      | CREDIT | Diners Club      |
-| discover         | CREDIT | Discover         |
-| jcb              | CREDIT | JCB              |
-| unionpay         | CREDIT | Union Pay        |
```

Comment:
It turns out that when a service searches using the `card_brand` parameter, they have to use on the strings in the leftmost column (even though the `"card_brand"` property in the response will use one of the strings in the rightmost column!) so maybe it’s best not to remove this table altogether for now.

Reply:
Thanks Alex - I'll add the table back in.

Reply:
Added back in - https://github.com/alphagov/pay-tech-docs/pull/330/commits/afd6d75278d6986dd07fb6343712593f270b7eaa